libfwupd/fwupd-request.c

```diff
@@ -101,6 +101,8 @@ fwupd_request_flag_to_string(FwupdRequestFlags flag)
 {
 	if (flag == FWUPD_REQUEST_FLAG_NONE)
 		return "none";
+	if (flag == FWUPD_REQUEST_FLAG_ALLOW_GENERIC)
```

Comment:
if you don't feel strongly, can this be `FWUPD_REQUEST_FLAG_ALLOW_GENERIC_MESSAGES`?

Reply:
We could, but wouldn't we want a generic icon/image at the same time?

Reply:
Oh...  OK never mind.

Reply:
How about we instead have two flags, one for message and one for icon?  Then the vendor can prefer the translated message but allow their own logo.

Reply:
Works for me, good idea.